L/libgsd/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgsd"
+version = v"3.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/glotzerlab/gsd/releases/download/v$(version)/gsd-$(version).tar.gz", "49a0e16276789b6953ece7e5a678bd22c128f8cc57cb967f4228ff6487770995")
```

Comment:
Since this is a git repo, it is better to use a `GitSource` here instead of an `ArchiveSource`.